src/models/User.js

```diff
@@ -57,5 +58,13 @@ const UserSchema = new mongoose.Schema({
   creditCardInfo: creditCardSchema,
 });
 
+//before we save the document we hash password; this points to the user (this= user)
+UserSchema.pre('save', async function () {
+  console.log(this.modifiedPaths()); //to see which properties were modified
+   console.log(this.isModified('name')); // in console we get false if we are not modifying name, and true if we do
```

Comment:
Do we need these logs here or was it just for debugging?

Reply:
It's for debugging only, should I delete them? as well as 
comments?